src/Zope2/utilities/mkwsgiinstance.py

```diff
@@ -227,7 +227,7 @@ def get_zope2path(python):
     try:
         output = subprocess.check_output(
             [python, '-c', 'import Zope2; print(Zope2.__file__)'],
-            universal_newlines=True,  # makes Python 3 return text, not bytes
+            text=True,  # makes Python 3 return text, not bytes
```

Comment:
```suggestion
            text=True,
```
We are on Python-3-only so this comment should no longer be needed.